pkg/controller/directvolumemigration/pvcs.go

```diff
@@ -0,0 +1,76 @@
+package directvolumemigration
+
+import (
+	"context"
+	corev1 "k8s.io/api/core/v1"
+	k8serror "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (t *Task) areSourcePVCsUnattached() error {
+	// This function provides state checking on source PVCs, make sure app is
+	// quiesced
+	return nil
+}
+
+func (t *Task) createDestinationPVCs() error {
```

Comment:
I wonder if this function essentially needs to run on all reconcile loops to make sure the destination PVCs are always present.